libtransmission/rpc-server.h

```diff
@@ -39,6 +39,7 @@ class tr_rpc_server
     {
     public:
         Settings() = default;
+        virtual ~Settings() = default;
```

Comment:
Can you change them for `{}` instead of `= default`? There are build issues on Apple clang 11.

Reply:
Marking this discussion as resolved since there's a bigger question in 6590 of supporting older compilers, and that issue existed before this PR